medical/tests/test_medical_patient.py

```diff
@@ -139,3 +139,20 @@ def test_check_birthdate_date(self):
                     now + timedelta(days=20),
                 )
             })
+
+    def test_search_age(self):
```

Comment:
When I log the search results, it gives me the domain that I have it returning in the original method: https://github.com/LasLabs/vertical-medical/pull/168/commits/d146aab01b45316ce9ffd98ea9bc88a8b9639aee#diff-13a235173691ae9abf959c2e1094800bR123 How do I get it to return the actual patient records?